adobecom/express
# [Frictionless] Limit Allowed Upload Types

As of now, no constraints exist on the SEO side for upload file types. After this PR, you should see that uploading pngs to a jpg->png QA would show an error toast message.

Resolves: https://jira.corp.adobe.com/browse/MWPW-158184

Test steps:

1. Go to a Frictionless QA page below
2. Drag and drop a file that doesn't have the right file type
3. A red error toast message should appear

Test URLs:
- After: https://frictionless-image-types--express--adobecom.hlx.live/express/feature/image/convert/png-to-jpg?martech=off Should not accept png
- https://frictionless-image-types--express--adobecom.hlx.live/express/feature/image/convert/jpg-to-png?martech=off Should not accept jpg
- https://frictionless-image-types--express--adobecom.hlx.live/express/feature/image/remove-background?martech=off should still accept all
